spec/autocomplete-manager-integration-spec.coffee

```diff
@@ -336,7 +335,7 @@ describe 'Autocomplete Manager', ->
               charMatch = editorView.querySelector('.autocomplete-plus li span.word .character-match')
               expect(word.textContent).toBe 'ab(c)c'
               expect(charMatch.textContent).toBe 'c'
-              expect(charMatch.parentNode).toHaveClass 'word'
+              expect(charMatch.parentNode.parentNode).toHaveClass 'word'
```

Comment:
If you are going to change the spec, check that the parent node is the snippet span class, not this parent.parent indirect check.